lib/ts/recipe/session/api/signout.ts

```diff
@@ -16,11 +16,12 @@
 import { send200Response } from "../../../utils";
 import { APIInterface, APIOptions } from "../";
 import { getSessionFromRequest } from "../sessionRequestFunctions";
+import { UserContext } from "../../../types";
 
 export default async function signOutAPI(
     apiImplementation: APIInterface,
     options: APIOptions,
-    userContext: any
+    userContext: UserContext
 ): Promise<boolean> {
     // Logic as per https://github.com/supertokens/supertokens-node/issues/34#issuecomment-717958537
 
```

Comment:
this is an unreleated change to mfa, but it's been pending since a while and it's a breaking change, so might as well do it here: In this api, we call getSession like the other APIs, and have sessionRequired: true.

Reply:
Done